### ISSUE TITLE
Multi_scan doesn't work if there's no .junit or report file already

### ISSUE THREAD

**Issue by zacoid55** [lyndsey-ferguson/fastlane-plugin-test_center#5]
I've run into a problem where I had an empty test_ouput folder and I ran multi_scan. It then said that there was no .junit xml report to then re-run anything from, even though the first run of multi_scan had made one

**Comment by lyndsey-ferguson** [comment]
Can you attach the log from the terminal when you execute your fastlane using `multi_scan`? Also, can you provide the options that you are passing to the action so I can duplicate the issue and fix it on my computer?

**Comment by zacoid55** [comment]
Steps to reproduce here:
I created a multi_scan lane which looked like this:
```lane :test_multiscan do
  multi_scan(
    workspace: 'MyWorkspace.xcworkspace',
    scheme: 'MyScheme'
  )
end```

I ran this with no `test_output` folder (which is where my report.junit will be. Then after the first scan I get this error:

```[09:58:14]: Error setting value './fastlane/test_output/report.junit' for option 'junit'
[09:58:14]: You passed invalid parameters to 'tests_from_junit'.
[09:58:14]: Check out the error below and available options by running `fastlane action tests_from_junit
[09:58:15]: Error: cannot find the junit xml report file './fastlane/test_output/report.junit'```

**Comment by lyndsey-ferguson** [comment]
I fixed the issue in v1.2.0; however, `multi_scan` will not re-scan failed tests as intended due to Issue #6. I am working on this and it should be released this week.

**Comment by zacoid55** [comment]
I've just updated to 1.2.0 will check right now :)

**Comment by zacoid55** [comment]
That fixed it, thanks @lyndsey-ferguson - you're an efficient wizard!

**Comment by lyndsey-ferguson** [comment]
@zacoid55 I won't be able to fix Issue #6 this week - not enough time before the Holidays. Hopefully next week!

**Comment by lyndsey-ferguson** [comment]
Issue #6 fixed in [v2.0.0](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/tree/v2.0.0)
